hivtrace/web/static/src/clusternetwork.js

```diff
@@ -2231,6 +2419,9 @@ var hivtrace_cluster_graph_summary = function (graph, tag) {
 
     if (!summary_table.empty()) {
         _.each (graph["Network Summary"], function (value, key) {
+        	if (_is_CDC_ && key == "Edges") {
```

Comment:
If this application were to become any larger, we would have to start considering multi-language support.